src/cli/commands/test/index.ts

```diff
@@ -394,56 +366,12 @@ function shouldFail(vulnerableResults: any[], failOn: FailOn) {
   return vulnerableResults.length > 0;
 }
 
-function isFixable(testResult: any): boolean {
-  return isUpgradable(testResult) || isPatchable(testResult);
-}
-
-function hasFixes(testResults: any[]): boolean {
-  return testResults.some(isFixable);
-}
-
-function isUpgradable(testResult: any): boolean {
-  if (testResult.remediation) {
-    const {
-      remediation: { upgrade = {}, pin = {} },
-    } = testResult;
-    return Object.keys(upgrade).length > 0 || Object.keys(pin).length > 0;
-  }
-  // if remediation is not available, fallback on vuln properties
-  const { vulnerabilities = {} } = testResult;
-  return vulnerabilities.some(isVulnUpgradable);
-}
-
-function hasUpgrades(testResults: any[]): boolean {
-  return testResults.some(isUpgradable);
-}
-
-function isPatchable(testResult: any): boolean {
-  if (testResult.remediation) {
-    const {
-      remediation: { patch = {} },
-    } = testResult;
-    return Object.keys(patch).length > 0;
-  }
-  // if remediation is not available, fallback on vuln properties
-  const { vulnerabilities = {} } = testResult;
-  return vulnerabilities.some(isVulnPatchable);
-}
-
-function hasPatches(testResults: any[]): boolean {
-  return testResults.some(isPatchable);
-}
-
-function isVulnUpgradable(vuln) {
-  return vuln.isUpgradable || vuln.isPinnable;
-}
-
-function isVulnPatchable(vuln) {
-  return vuln.isPatchable;
+function validateSeverityThreshold(severityThreshold) {
+  return SEVERITIES.map((s) => s.verboseName).indexOf(severityThreshold) > -1;
```

Comment:
Could use `includes` here

```suggestion
  return SEVERITIES.map((s) => s.verboseName).includes(severityThreshold);
```

Reply:
To be honest, I just moved these as is. I shall make a note and can do another change.